cloud_recording.go

```diff
@@ -47,14 +47,16 @@ type (
 	// CloudRecordingMeeting represents a zoom meeting object
 	CloudRecordingMeeting struct {
 		UUID           string          `json:"uuid"`
-		ID             string          `json:"id"`
+		ID             int             `json:"id"`
```

Comment:
Why did you make this change? According to the documentation, it is a string. Same with `TotalSize` below. If these only ever return integers, can you please contact Zoom to update their documentation? (Their dev relations team is usually pretty responsive.) Then we can keep the update here.

<img width="663" alt="Screen Shot 2021-05-07 at 10 37 22 AM" src="https://user-images.githubusercontent.com/1058475/117487750-57b18c80-af20-11eb-9945-7e02574861a0.png">